corehq/apps/analytics/static/analytix/js/hubspot.get_demo.js

```diff
@@ -0,0 +1,41 @@
+/* global window, hbspt */
```

Comment:
Nitpick, but periods in filenames are unconventional, I'd just call this `hubspot_get_demo.js`

Reply:
this file is getting removed in https://github.com/dimagi/commcare-hq/pull/22503